### ISSUE TITLE
Check count in tab before declaring folder empty

### ISSUE THREAD

**Issue by jidanni** [roundcube/roundcubemail#9554]
### Prerequisites

- [X] I have [searched](https://github.com/roundcube/roundcubemail/issues?q=is%3Aissue) for duplicate or closed feature requests
- [X] I have [searched](https://plugins.roundcube.net/) for plugins that provide already provide the feature

### Proposal

I have an idea that will cure Roundcube of the "false empty mailbox bug."

For instance,

![m](https://github.com/user-attachments/assets/3847834d-6c4f-47b2-b61b-c33500e54eb4)

We observe Roundcube insists that this folder is empty.

Ah, but on the top of the screen (see red arrow), it is revealed the 9109 count!

And on #9553 we see that 9109 refers indeed to this exact folder!

Therefore, all Roundcube needs to do, is before declaring the folder as empty, first check if it also intends to put a number into the browser tab. If this is the case, it should trigger an automatic reload, which will then cause the page to be normal. Thanks!

In summary, some parts of Roundcube know the correct count, so other parts  of Roundcube should consult that correct part, before reporting a wrong count to the user.

### Motivation and context

Because seeing that ones Inbox has been "wiped out" might cause a heart attack.

**Comment by alecpl** [comment]
It never happened to me, so it would be better to find what the underlying issue is than to create workarounds.

First, you have to use most recent version. Second, what are the steps to reproduce the issue. Then, what is in browser console and Roundcube logs. Finally `imap_debug` log might be useful.

**Comment by jidanni** [comment]
It is certainly some race condition. Something about starting too fast.
If it happens it usually happens the first time the browser connects to the server in a day.

Another factor is I am on a slow internet connection.

Alas as I am a mere end user... at least next time it happens I can check the browser console...
